crypto/bls_no_relic.go

```diff
@@ -56,6 +56,22 @@ func IdentityBLSPublicKey() PublicKey {
 	panic(relic_panic)
 }
 
```

Comment:
suggesting to fix a typo in line 12: https://github.com/onflow/flow-go/blob/4280825d60e9c683aaa31caeb630eeaaca3a3efc/crypto/bls_no_relic.go#L12
change to: 'functional_**i**_ties'